lighthouse-core/test/report/v2/renderer/details-renderer-test.js

```diff
@@ -95,31 +81,6 @@ describe('DetailsRenderer', () => {
       assert.equal(items.children.length, 3, 'did not render children');
     });
 
-    it('renders nested structures', () => {
```

Comment:
actually, without the block part it's just a duplicate of the `renders lists with headers` test above, so it's probably fine to remove and get more detailed list tests in #2019